### ISSUE TITLE
Facilities Team Refinement

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#16686]
# Q2 priorities: 
Tickets for refining. Lists do not include previously refined tickets.

## 1) V3 component adoption - May 3 deadline - department-of-veterans-affairs/va.gov-cms#17653
* Tickets complete, deadline met unless bugs arise after May 3 cutover
- #18126

## 2) va-icon adoption - end of May deadline - department-of-veterans-affairs/va.gov-cms#17722
  - 🟢 FE work is refined
  - 🟡 Encountering small blockers along the way

## 4) Lighthouse V1 / FL update to adopt - currently, June 1 deadline - department-of-veterans-affairs/va.gov-cms#14507
- Our ticket [15344](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15344) should be resolved by [IIR-#540](https://github.com/department-of-veterans-affairs/va-iir/issues/540)
- [ ] #18183 - waiting on IIR confirmation they'll handle

## 5) Service Location Enhancement launch blockers - department-of-veterans-affairs/va.gov-cms#16825
- 🟢  Launch is complete
- [ ] Product: #18145 - Jill / Dave / Michael to refine async

## 6) VBA Staging Review Blockers - department-of-veterans-affairs/va.gov-cms#16268
- [ ] Team: #18159 

### VBA Pilot remainders - department-of-veterans-affairs/va.gov-cms#10514
- [ ] FE/UX: #17966 

## 7) VA Health Chat and VA Health Connect - department-of-veterans-affairs/va.gov-cms#15037
- 🟡 Blocked on VHA DM 
- UX: department-of-veterans-affairs/va.gov-cms#16009  -- not critical to refine now, bc this initiative is delayed.

## 8) FL Provider Core Training search/ filters - department-of-veterans-affairs/va.gov-cms#14225
- Need to scrub the epic for next steps tickets to refine

## 10) Vet Center inline editor / allow only one / winnower - department-of-veterans-affairs/va.gov-cms#17708 / department-of-veterans-affairs/va.gov-cms#7818
These tickets are not in priority order yet. We may want to do a product level kickoff before we try to refine them. 
- [ ] Drupal: department-of-veterans-affairs/va.gov-cms#17674
- [ ] CM: department-of-veterans-affairs/va.gov-cms#17733 
- [ ] Drupal: department-of-veterans-affairs/va.gov-cms#17676
- [ ] Drupal & FE: department-of-veterans-affairs/va.gov-cms#17681. -- need some addl Drupal specifics in order to assess FE impact
- [ ] Drupal: #18003 - this is the launch step, and needs re-refined when other epic tickets are done
- [ ] Drupal: department-of-veterans-affairs/va.gov-cms#5981 - icebox? 
- [ ] Drupal: department-of-veterans-affairs/va.gov-cms#15124 - icebox?
- [ ] Drupal: department-of-veterans-affairs/va.gov-cms#15125 - icebox?

## 11) Vet Centers design iteration (2.0) - department-of-veterans-affairs/va.gov-cms#17661
- 🟡 Need Product work to get the Epic in shape re: scope after Jordan's research review, plan out next steps work, etc.

## 13) Mobile VAMC design - department-of-veterans-affairs/va.gov-cms#12923
- 🟡 When Amanda has concluded research review, need to revisit UX scope to cut Research, Design & potentially Content Modeling tickets. (Research review has no ticket. Thoughts?)

## 14) VBA Regional Office National MVP - department-of-veterans-affairs/va.gov-cms#14477
- Will create new Editor research tickets as needed.
- Need to review issues in epic for whether they qualify as bug fixes found during launch / training cycles, or as enhancements that push to post-MVP epic.

## 18) Facility Locator: "Service type" field iterations - [#15541](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15541)
- DOUBLE CHECK for more tickets to be refined

## 20) Improve services accordions & locations across products - department-of-veterans-affairs/va.gov-cms#17350

- [ ] Drupal: department-of-veterans-affairs/va.gov-cms#17380
- [ ] #17974
- [ ] Drupal: VAMC/VBA: department-of-veterans-affairs/va.gov-cms#17497

## Other Q2 priorities
- 🟡 MHV on VAMC sites [#12942](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12942) -- no product details yet
- 🟡 Police transparency MVP [#14327](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14327) -- generally: blocked on Axon data next steps
    - [ ] Drupal: department-of-veterans-affairs/va.gov-cms#16813 - can be refined
- FL plain language services search [#15541](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15541)
    - [ ] department-of-veterans-affairs/va.gov-cms#14993

# Other work (not Q2 prioritized)
#18031 

## Other 
- [ ] Analytics: department-of-veterans-affairs/va.gov-cms#16389
- [ ] Drupal/UX: VAMC: department-of-veterans-affairs/va.gov-cms#17397
- [ ] FE: department-of-veterans-affairs/va.gov-cms#17836 - repros after typography merges?
- [ ] Phone number epic #17854 - contains several Drupal & FE tix that need refining
    - [ ] Drupal: #17863

**Comment by mmiddaugh** [comment]
Additional candidate for this (or the next) refinement: 
- [#16148  Update phone numbers in Facilities products to use component](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16148#top)

**Comment by xiongjaneg** [comment]
@xiongjaneg create launch ticket for this change

**Comment by mmiddaugh** [comment]
@laflannery found these accessibility defects - which has the highest value? 
- FE: Update phone numbers in Facilities products to use component #16148
- FE: Updated Lovell switch alert type should not appear on the page table of contents  #15569
- FE: VAMC event, story, and news release pagination is not announced properly #12089

@xiongjaneg suggestions for group discussion: 
- Standardize spotlight patterns in CMS facility content types #13804
- Remove Featured field order from featured fieldset #13688

**Comment by laflannery** [comment]
@mmiddaugh I would say the Lovell switcher we should tackle first, that might take some creativity. The other 2 are related to the overall component work we are doing so those would start that work of getting everything updated properly

**Comment by xiongjaneg** [comment]
@laflannery For #13804, is that one ready for pre-refinement for Front End or do you want to discuss in UX sync first?

**Comment by laflannery** [comment]
@xiongjaneg It's ready but it's not an engineering ticket, it's a UX ticket to standarize/create the pattern, then from that would be follow up engineering tickets

**Comment by xiongjaneg** [comment]
Thanks @laflannery that's what I thought. @davidmpickett could we get #13804 on the agenda for next UX sync. Thank you!

**Comment by mmiddaugh** [comment]
### Related issues
- Should we include [Front end changes and associated tests for Address module upgrade to version 2 #17193](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/17193) along with [Drupal: back end changes and associated tests for Address module upgrade to version 2 #17191](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17191)?

### Priority front end tasks
**Front end pre-refinement**
- VAMC: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17237
(suggest replacing with https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12079 - an a11y-defect-2)
- Defect: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15656
(suggest replacing with https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15390 - identified VHA DM priority)
- [#17107
 Front end: Implement Lovell Cerner (Oracle Health) changes](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/17107) 
  - Can we please confirm that no tests should be added/updated with the Cerner/Oracle health change? 
  - Do we need an issue for testing the implementation in staging? 
### Other considerations
- **Drupal**
   - [[CONSIDER] Temporary location override #17144](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/17144)
- **Front end pre-refinement**
  - [Fix inconsistent logic in VAMC pages display of Top task links](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/15567) is somewhat related to [Lovell Tricare URL changes #17299](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17299)

**Comment by davidmpickett** [comment]
I don't know that #15608 is actionable. We haven't implemented that feature, it's been on hold.

**Comment by jilladams** [comment]
#17566 was removed but will block VBA staging review. Dave to keep refining and we'll come back to it.

**Comment by davidmpickett** [comment]
@jilladams FYI - I added sections here for Service Location and VBA launch blocking tickets.

**Comment by jilladams** [comment]
Ticket's been expanded to be full team, so includes UX tickets as well.

**Comment by jilladams** [comment]
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17791 need to add for FE

**Comment by jilladams** [comment]
Removing 2 sections per updates to Q2 priorities: 

## 12) Multiple photos for facility pages - department-of-veterans-affairs/va.gov-cms#15718
- [ ] UX: department-of-veterans-affairs/va.gov-cms#16719
- [ ] UX: department-of-veterans-affairs/va.gov-cms#17254
- [ ] UX: department-of-veterans-affairs/va.gov-cms#16890

## 15) Featured Stories rethink - #11401
- [ ] UX: department-of-veterans-affairs/va.gov-cms#13914 - is this the right ticket to start UX thinking?
- [ ] CM: May need a ticket, tbd
- [ ] Drupal & UX: department-of-veterans-affairs/va.gov-cms#13695 - may be a good starting point to review options? Or: we may not want to bias process / may want to start greenfield with UX.

**Comment by mmiddaugh** [comment]
@jilladams @davidmpickett 
For refinement consideration:
- Re: Facility Locator: "Service type" field iterations: I believe there is some design layout work to implement progressive disclosure for the Service type field following meetings with the Design system team - this may not yet be ticketed. 

- [FE: [VAMC] Veteran-facing navigation experience between system and facility level health service #13910](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/13910)- this is a VHA priority, would be good to validate my assumption that this is a small lift and have it ready for a future sprint

- Drupal work to move to implement separate phone number and extension fields

**Comment by aklausmeier** [comment]
> Re: Facility Locator: "Service type" field iterations: I believe there is some design layout work to implement progressive disclosure for the Service type field following meetings with the Design system team - this may not yet be ticketed. 

#17835 - related ticket has been created

**Comment by davidmpickett** [comment]
Noting that priority 19 has been removed from last as that is being transferred to IIR
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17748

**Comment by davidmpickett** [comment]
> @jilladams @davidmpickett For refinement consideration:
> 
> * Re: Facility Locator: "Service type" field iterations: I believe there is some design layout work to implement progressive disclosure for the Service type field following meetings with the Design system team - this may not yet be ticketed.

#17835 is refined and candidate for Sprint 3

> * [FE: [VAMC] Veteran-facing navigation experience between system and facility level health service #13910](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/13910)- this is a VHA priority, would be good to validate my assumption that this is a small lift and have it ready for a future sprint

I updated the ticket to reflect the new scope and added it to list here for refinement. 

> * Drupal work to move to implement separate phone number and extension fields

Phone number epic is at the bottom of the list and I added the ticket from that epic that Jill, Laura, and I identified as a quick win as a first one to refine.

**Comment by Agile6MSkinner** [comment]
@jilladams are you ready to move on from this ticket?

**Comment by jilladams** [comment]
Have goldfish, ready to travel wherever you're headed. Closing.
